lib/jekyll/commands/new.rb

```diff
@@ -57,53 +67,29 @@ def initialized_post_name
 
         private
 
-        def gemfile_contents
-          <<-RUBY
-source "https://rubygems.org"
-ruby RUBY_VERSION
-
-# Hello! This is where you manage which Jekyll version is used to run.
-# When you want to use a different version, change it below, save the
-# file and run `bundle install`. Run Jekyll with `bundle exec`, like so:
-#
-#     bundle exec jekyll serve
-#
-# This will help ensure the proper Jekyll version is running.
-# Happy Jekylling!
-gem "jekyll", "#{Jekyll::VERSION}"
-
-# This is the default theme for new Jekyll sites. You may change this to anything you like.
-gem "minima"
-
-# If you want to use GitHub Pages, remove the "gem "jekyll"" above and
-# uncomment the line below. To upgrade, run `bundle update github-pages`.
-# gem "github-pages", group: :jekyll_plugins
-
-# If you have any plugins, put them here!
-group :jekyll_plugins do
-   gem "jekyll-feed", "~> 0.6"
-end
-RUBY
-        end
-
         def create_site(new_blog_path)
-          create_sample_files new_blog_path
+          create_static_files new_blog_path
```

Comment:
Again I seemed rude without intending to be.  We've already tied into this API a bit, which means we expect it to remain stable, and that also means that I would assume other users might have already tied into it and expect it to remain stable.

API changes like this should be left to major versions rather than minor versions.